siliconcompiler/libs/asap7sc7p5t.py

```diff
@@ -14,24 +14,28 @@ def _setup_lib(chip, libname, suffix):
                'fast': 'ff',
                'slow': 'ss'}
 
-    libdir = os.path.join('..', 'third_party', 'pdks', foundry, process, 'libs', libname, rev)
+    libdir = os.path.join('third_party', 'pdks', foundry, process, 'libs', libname, rev)
 
     # todo: remove later
     lib.set('option', 'pdk', 'asap7')
 
     # timing
     for corner_name, lib_corner in corners.items():
         lib.add('output', corner_name, 'nldm',
-                libdir + '/nldm/' + libname + '_' + lib_corner + '.lib.gz')
+                libdir + '/nldm/' + libname + '_' + lib_corner + '.lib.gz',
+                dependency='siliconcompiler_data')
```

Comment:
It might be nice to extend the Library/PDK classes to automatically attach the package information if not declared.

Reply:
If a pdk just is allowed to have one dependency then that sounds like a good idea.

Reply:
Adding it like this `PDK(chip, name, dependency='siliconcompiler_data')`.